adapter/repository/in-memory-simple/src/test/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepositoryTest.java

```diff
@@ -290,6 +314,24 @@ void getCalendarByServiceIdShouldReturnRelatedCalendar() {
         assertEquals(mockCalendar01, underTest.getCalendarByServiceId("service id01"));
     }
 
+    @Test
+    void getCalendarAllShouldReturnCalendarCollection() {
+        final Calendar mockCalendar00 = mock(Calendar.class);
+        final Calendar mockCalendar01 = mock(Calendar.class);
+        final InMemoryGtfsDataRepository underTest = new InMemoryGtfsDataRepository();
+
+        when(mockCalendar00.getServiceId()).thenReturn("service id00");
+        when(mockCalendar01.getServiceId()).thenReturn("service id01");
+
+        underTest.addCalendar(mockCalendar00);
+        underTest.addCalendar(mockCalendar01);
+
+        final Map<String, Calendar> toCheck = underTest.getCalendarAll();
+        assertEquals(2, toCheck.size());
+        assertTrue(toCheck.containsKey("service id00"));
+        assertTrue(toCheck.containsKey("service id01"));
+    }
+
     @Test
     public void addSameTripTwiceShouldReturnNull() {
```

Comment:
shall the public modifier also be removed here? Is it present on any other test method?

Reply:
We actually have this modifier in some tests, I tend to remove that when I encounter one.